### PR TITLE
Preparations for release 0.2.0

### PR DESCRIPTION
**Pull request by kaizimmerm**
* Promote hawkBit example app to runtime
* Final patch round before release:
  * Vaadin 7.7.7
  * Spring Boot 1.4.5
  * Spring Cloud Camden.SR6
  * Flyway 4.0.3
* CQs are now listed in a more readable format

### CODE REVIEW

File: hawkbit-runtime/pom.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+   <packaging>pom</packaging>
+
+   <distributionManagement>
+      <repository>
+         <id>repo.eclipse.org</id>
+         <name>hawkBit Repository - Release</name>
+         <url>https://repo.eclipse.org/content/repositories/hawkbit-releases</url>
+      </repository>
+      <snapshotRepository>
+         <id>repo.eclipse.org</id>
```
Comment (SirWayne) [inline hawkbit-runtime/pom.xml]: Fix id and name and use e.g. snapshot repo?

File: 3rd-dependencies/generateReleaseMd.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (michahirsch) [inline 3rd-dependencies/generateReleaseMd.sh]: The `listDeps.sh` contains a license header, probably also need to add here?

File: docs/src/main/resources/documentation/guide/runhawkbit.md [hunk 3]
```diff
@@ -7,35 +7,33 @@ title: Run hawkBit
...
+- You have a working [hawkBit examples build](https://github.com/eclipse/hawkbit/examples).
 
 # Steps
 
 ## Adapt hawkBit Update Server and Device Simulator to your environment.
 
 As mentioned you can create your own application with _hawkBit_ inside or adapt the existing example app. The second option will be shown here.
 
-### Set MariaDB dependency to compile in the [example App POM](https://github.com/eclipse/hawkbit/blob/master/examples/hawkbit-example-app/pom.xml)
+### Set MariaDB dependency to compile in the [example App POM](https://github.com/eclipse/hawkbit/blob/master/hawkbit-runtime/hawkbit-update-server/pom.xml)
```
Comment (michahirsch) [inline docs/src/main/resources/documentation/guide/runhawkbit.md]: `[example App POM]` -> `[update-server POM]`?

File: README.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ https://hawkbit.eu-gb.mybluemix.net/UI/
 
 # Compile, Run and Getting Started
 
-We are not providing an off the shelf installation ready hawkBit update server. However, we recommend to check out the [Example Application](examples/hawkbit-example-app) for a runtime ready Spring Boot based update server that is empowered by hawkBit. In addition we have [guide](https://github.com/eclipse/hawkbit/wiki/Run-hawkBit) for setting up a complete landscape.
+We are not providing an off the shelf installation ready hawkBit update server. However, we recommend to check out the [Example Application](examples/hawkbit-example-app) for a runtime ready Spring Boot based update server that is empowered by hawkBit. In addition we have [guide](http://www.eclipse.org/hawkbit/documentation/guide/runhawkbit.html) for setting up a complete landscape.
```
Comment (michahirsch) [inline README.md]: > However, we recommend to check out the [Example Application](examples/hawkbit-example-app)

I guess we would write here that we do offer a hawkbit-runtime update-server for easy to startup an hawkBit server, ready to use, which can be customized by your needs, right?

**Pull request merged**